### DISCUSSION TITLE
Subscribing (+Tipping) could use some positive feedback

### DISCUSSION THREAD

**Discussion by Cwpute** [funmusicplace/mirlo#849] [Ideas]
I just subscribed to Mirlo, and after the process went through i was a bit underwhelmed. At first i didn't notice anything that was changed, but then noticed the "Subscribe" button was just replaced with a boring and off-centered text that simply read `Subscribed`.
A simple toast message to confirm your subscription, and perhaps an "unoff-centered", more visible acknowledgment of your subscription in place of the subscribe button on the artist page would be enough :) we don't want the website to become a positive-feedback-festival with dynamic text and colours and animations everywhere for every action performed.

Can't say for the action of Tipping, but it probably is similar !

**Comment by simonv3** [comment]
Yeah generally all of our purchasing could use some cute "yay" text.

**Reply by Cwpute** [reply]
Artists could even personalize their own "yay" text if they want ?

**Comment by Cwpute** [comment]
Just tipped towards a crowdfunding goal on a french website, which showed a cute short gif of one of the developers : 
![image](https://github.com/user-attachments/assets/815d3876-6b8c-46a1-91f7-7b18f0d27bf3)
Really goes towards making you feel like you actually did something cool.
